pkg/backend/filestate/backend.go

```diff
@@ -72,17 +80,46 @@ func (r localBackendReference) Name() tokens.QName {
 }
 
 func IsLocalBackendURL(url string) bool {
```

Comment:
Should this function renamed? Having a IsLocalBackenURL return true for cloud storage, sounds unexpected

Reply:
I've renamed it to `IsFileStateBackendURL`. I've also replaced the `localBackendURL` constant with a `supportedSchemas` slice.